pages/blog/posts/GSOD-docs-project-2024.md

```diff
@@ -0,0 +1,46 @@
+    byline: Technical writer & Community Manager
+excerpt: 'An announcement blog post to share the JSON Schema documentation efforts from the GSOD team.'
+---
+
+## Introduction
+
+For three years, I've eagerly followed the Google Season of Docs (GSoD), and this year, I had two organizations in mind. Midway through the application process, one of the organizations didn't make it past Google's screening, so I decided to focus on JSON Schema. I worked on my proposal and made it through the interview, and it's been an amazing journey contributing to JSON Schema's documentation alongside my team members: [Valeria Hernandez](https://www.linkedin.com/in/valeriahhdez/), [Blessing Ene Anyebe](https://www.linkedin.com/in/anyebe-blessing-ene-kwennb/), [Dhairya Amrish Majmudar](https://www.linkedin.com/in/dhairya-majmudar/), and [Benjamin Granados](https://www.linkedin.com/in/benjagranados/).
+
+## The Project
+
+The main goal of JSON Schema's [documentation project](https://github.com/orgs/json-schema-org/discussions/687) is to create a scalable documentation architecture that is clear, consistent, and comprehensive. To achieve this through GSoD, we focused on:
+
+- **Restructuring the specification docs**: This section needs a better structure to provide easy access to the latest specification versions. Users need to understand how JSON Schema has evolved across different releases and the features introduced in each version. Restructuring this section makes it more useful for all the JSON Schema [personas](https://github.com/json-schema-org/community/issues/336#personas).
+- **Restructuring the reference documentation**: We are creating a new version of the reference documents, using modern documentation principles, such as the [diataxis framework](https://diataxis.fr/), to improve the existing "Understanding JSON Schema" book. This will provide a more efficient onboarding experience and better content organization.
+
+## The Update
+
+Since May, the team has been working hard to achieve these goals. We've conducted extensive research, mapped out [developer experience journeys](<(https://www.figma.com/board/TRjQUw33K93y8RlJMSRkJs/JSON-dev-journey?node-id=0-1&t=gYkdGtHk2sKrCQM0-0)>), implemented the [keywords page](https://json-schema.org/understanding-json-schema/keywords), and added on-page tables of content. We've also introduced [Next-Previous navigation](https://github.com/json-schema-org/website/pull/807) sections and restructured the [specification docs](https://github.com/json-schema-org/website/pull/823).
+
+What's pending? We're currently working on the [JSON Hyper-Schema](https://github.com/json-schema-org/website/issues/896) and [Migration pages](https://github.com/json-schema-org/website/issues/897) as part of the specification docs. Additionally, we're drafting style guides and creating [overview content buckets](https://github.com/json-schema-org/website/issues/803) to enhance the documentation.
```

Comment:
@valeriahhdez I think we need to add to this paragraph more details about what is pending from your side as this reflects mostly the pending work from Blessing's side.